src/main/java/uk/ac/ebi/eva/commons/mongodb/services/VariantSourceService.java

```diff
@@ -20,7 +20,7 @@
         return convert(repository.findAll());
     }
 
-    private List<VariantSource> convert(List<VariantSourceMongo> variantSources) {
```

Comment:
Not sure if you can create a repository from a interface, Although you can create a projection from an interface.